polynote-frontend/polynote/ui.js

```diff
@@ -801,6 +802,16 @@ class EditBuffer {
     }
 }
 
+function maxId(cells) {
```

Comment:
I think we can get the max of an array using `Math.max` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max#Using_Math.max()